CONTRIBUTING.md

```diff
@@ -0,0 +1,145 @@
+# Contributing Guidelines <!-- omit in toc -->
+
+Thanks for taking the time to contribute! We would like the Astuto community to be a welcoming environment for both seasoned programmers and absolute beginners. The following guidelines should help you get started regardless of your skill level.
+
+- [How to contribute](#how-to-contribute)
+  - [Reporting bugs](#reporting-bugs)
+  - [Proposing features](#proposing-features)
+  - [Coding](#coding)
+  - [Getting involved in the community](#getting-involved-in-the-community)
+- [Styleguides](#styleguides)
+  - [Git commit messages](#git-commit-messages)
+  - [Coding conventions](#coding-conventions)
+    - [General coding conventions](#general-coding-conventions)
+    - [Ruby coding conventions](#ruby-coding-conventions)
+    - [JavaScript coding conventions](#javascript-coding-conventions)
+- [This is my first contribution! Help!!](#this-is-my-first-contribution-help)
+  - [Workflow](#workflow)
+  - [Look for label "good first issue"](#look-for-label-good-first-issue)
+  - [Any doubt?](#any-doubt)
+- [Community](#community)
+  - [Code of Conduct](#code-of-conduct)
+  - [Language](#language)
+  - [Want to chat?](#want-to-chat)
+- [Credits](#credits)
+
+## How to contribute
+
+There are various ways to contribute to Astuto. In this section we list the most common ones.
+
+### Reporting bugs
+
+[Open a new issue](https://github.com/riggraz/astuto/issues/new) on GitHub. In doing so, please follow these guidelines:
+
+- Check whether this bug has been reported before in the [issues panel](https://github.com/riggraz/astuto/labels/bug). If that's the case, consider adding a comment to the already open issue. Otherwise, open a new issue.
+- Choose a short, meaningful title
+- Start the issue description with a quick summary of the problem you encountered
+- Include steps to reproduce the issue
+- State how you expected the program to behave, and how it actually behaved
+- If applicable, write down stuff you tried to solve the problem that didn't work
+- Feel free to include personal thoughts about why you think this issue is occurring
+
+### Proposing features
+
+As for bug reports, we use GitHub issues to track feature requests. So, if you want to propose a feature or enhancement, just [open a new issue](https://github.com/riggraz/astuto/issues/new). As always, check whether that feature has already been proposed before posting.
```

Comment:
Maybe we could add that if the feature has already been posted, people can react to it (with a 👍. for example) to show their desire to have it too ?